File: M8/Exemplos/App_eXperience_Java/app/src/main/java/com/elgin/elginexperience/shipay/models/wallet/WalletForm.java
package com.elgin.elginexperience.shipay.models.wallet;

import com.google.gson.annotations.SerializedName;

public class WalletForm {
    @SerializedName("items")
    public WalletItems items;
    @SerializedName("order_ref")
    public WalletOrderRef order_ref;
    @SerializedName("total")
    public WalletTotal total;
    @SerializedName("wallet")
    public WalletInfo wallet;
}

File: M8/Exemplos/App_eXperience_Kotlin/app/src/main/java/com/example/e1_elgin_kotlin/shipay/models/wallet/WalletForm.kt
package com.example.e1_elgin_kotlin.shipay.models.wallet

import com.google.gson.annotations.SerializedName

data class WalletForm (
    @SerializedName("items")
    var items: WalletItems,

    @SerializedName("order_ref")
    var order_ref: WalletOrderRef,

    @SerializedName("total")
    var total: WalletTotal,

    @SerializedName("wallet")
    var wallet: WalletInfo
)
